english/INDIA/for_github

```diff
@@ -0,0 +1 @@
+hindi is the national language of INDIA!!
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.